bluemira/magnetostatics/finite_element_2d.py

```diff
@@ -50,7 +50,7 @@ def Bz_coil_axis(
     return 0.5 * MU_0 * current * r**2 / (r**2 + (pz - z) ** 2) ** 1.5
 
 
-def _convert_const_to_dolfin(value: float):
+def _convert_const_to_dolfin(value: float) -> dolfin.Constant:
```

Comment:
This is correct but we're just about to change a lot about dolfin stuff and this might kick up a few conflicts
```suggestion
def _convert_const_to_dolfin(value: float):
```